arch/risc-v/include/bl808/irq.h

```diff
@@ -31,10 +31,18 @@
 
 /* Map RISC-V exception code to NuttX IRQ */
 
-#define NR_IRQS (RISCV_IRQ_SEXT + 57)
+#define NR_IRQS (RISCV_IRQ_SEXT + 114)
```

Comment:
Maybe you could include a comment in the code to explain why it was moved from 57 to 114. Although that information is in the commit message, it is better to get more info directly in the source code.